guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,478 @@
 {
```

Comment:
<p style="user-select: auto;">Workforce and tracker headings has indentation issues</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/684/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<h5>Adjusted the <a href="https://developers.arcgis.com/python/api-reference/arcgis.apps.tracker.html" target="_blank">tracker</a> to match the <a href="https://developers.arcgis.com/python/api-reference/arcgis.apps.workforce.html" target="_blank">workforce</a></h5><p> </p>